app.json

```diff
@@ -13,6 +13,9 @@
     },
     {
       "url": "https://github.com/heroku/heroku-buildpack-pgbouncer"
+    },
+    {
+      "url": "https://github.com/heroku/heroku-buildpack-nginx"
```

Comment:
is this the only thing necessary to run NGINX on heroku?
Is the `config/nginx.conf` common to both heroku and docker